### PR TITLE
Editable table

### PR DESCRIPTION
**Pull request by jonathan-mejia-deluna**
fixing button removal when changing receipt file uploaded

### CODE REVIEW

**Review by orogenic — APPROVED** [review]

**Comment by orogenic** [comment]
idk what you did to your commit structure exactly but this 1 line diff should not require a separate merge commit. try to put it in a single commit on top of main instead of whatever this is.

File: static/transaction.js [hunk 1]
```diff
@@ -63,7 +63,6 @@ function makeCellEditable(event, inputAttributes, handler) {
 
     if (handler === receiptEdit) {
         var button = td.children[1];
-        console.log(button.name);
```
Comment (orogenic) [inline static/transaction.js]: you should not do things like this to the commit history. you can remove this line from the previous commit and avoid this merge commit.

File: static/transaction.js [hunk 2]
```diff
@@ -62,7 +62,8 @@ function makeCellEditable(event, inputAttributes, handler) {
     td.removeEventListener("click", handler);
 
     if (handler === receiptEdit) {
-        var button = document.querySelector("td.receipt button");
+        var button = td.children[1];
+        console.log(button.name);
```
Comment (orogenic) [inline static/transaction.js]: the line in question to remove

File: static/transaction.js (hunk 1 above)
Comment (jonathan-mejia-deluna) [reply static/transaction.js]: I think it was because I resolved the conflict on Github. The alternative is to merge editableTable into master locally?

**Pull request merged**